akka-stream/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -160,6 +160,9 @@ ProblemFilters.exclude[Problem]("akka.stream.StreamRefMessages*")
 # #27266 changes to streams internals
 ProblemFilters.exclude[Problem]("akka.stream.impl.*")
 
+# Deprecates status message based api #27503
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.scaladsl.Sink.actorRefWithAck$default$5")
```

Comment:
Hmm, deprecating shouldn't have ended up with a binary incompability, should it?